data.sql

```diff
@@ -84,4 +84,128 @@ BEGIN;
 UPDATE animals.animals
 SET owners_id = 'Dean Winchester'
 WHERE name IN ('Angemon', 'Boamon');
-COMMIT;
\ No newline at end of file
+COMMIT;
+
+-- insert records into vets table
+INSERT INTO vets(name, age, date_of_graduation) VALUES('William Tatcher',45,'23/04/2000');
+INSERT INTO vets(name, age, date_of_graduation) VALUES('Maisy Smith',26,'17/07/2019');
```

Comment:
- [ ] Kindly insert the correct graduation date in `Maisy Smith`, It is `'2019-01-17'`